eng/Versions.props

```diff
@@ -152,11 +152,11 @@
     <Tooling_MicrosoftCodeAnalysisBannedApiAnalyzersPackageVersion>$(Tooling_MicrosoftCodeAnalysisAnalyzersPackageVersion)</Tooling_MicrosoftCodeAnalysisBannedApiAnalyzersPackageVersion>
     <Tooling_RoslynDiagnosticsAnalyzersPackageVersion>$(Tooling_MicrosoftCodeAnalysisAnalyzersPackageVersion)</Tooling_RoslynDiagnosticsAnalyzersPackageVersion>
     <Tooling_MicrosoftVisualStudioLanguageServicesPackageVersion>$(MicrosoftVisualStudioLanguageServicesPackageVersion)</Tooling_MicrosoftVisualStudioLanguageServicesPackageVersion>
-    <XunitAnalyzersPackageVersion>0.10.0</XunitAnalyzersPackageVersion>
-    <XunitCombinatorialPackageVersion>1.4.1</XunitCombinatorialPackageVersion>
-    <XunitStaFactVersion>1.1.11</XunitStaFactVersion>
-    <XunitVersion>2.4.2</XunitVersion>
-    <XunitExtensibilityExecutionPackageVersion>$(XunitVersion)</XunitExtensibilityExecutionPackageVersion>
+    <XUnitAnalyzersVersion>1.7.0</XUnitAnalyzersVersion>
```

Comment:
Turns out Arcade defines a bunch of variables for xunit versions, and adds implicit references to the packages, so in order for the SDK to not complain about version mismatches, we have to use the same variable names, hence the rename here.